WLC/benchmark.py

```diff
@@ -30,13 +30,16 @@ def benchmark_file(file_name):
     code = code.lower()
     fixed_code = CodeFixer(code, indents).fix()
 
-    difference = editdistance.eval("".join(fixed_code.split()), "".join(expected_code.split()))
+    difference = editdistance.eval("".join(code.split()), "".join(expected_code.split()))
+    difference_fixed = editdistance.eval("".join(fixed_code.split()), "".join(expected_code.split()))
 
     length = len("".join(expected_code.split()))
     accuracy = round(100 - (difference * 100 / length))
+    accuracy_fixed = round(100 - (difference_fixed * 100 / length))
 
-    LOGGER.info('Accuracy: {}%, File: {}'.format(accuracy, file_name))
-    return accuracy, length
+    LOGGER.info('Accuracy w/o fixing: {}%, Accuracy w/ fixing: {}%, Fix improvement: {}%,  File: {}'.format(
```

Comment:
Just a small suggestions, instead of having it like
`Accuracy w/o fixing: {}, Accuracy w/ fixing: {}, Fix improvement: {}%, File: {}`
can we have something like:
```
File: {}\n
  Accuracy:\n
    w/o fixing: {}%\n
    w/ fixing: {}%\n
    change: {}%
```

Reply:
The length of the line isnt that long and its more compact this way.